packages/mds-agency/request-handlers.ts

```diff
@@ -329,7 +330,11 @@ export const submitVehicleEvent = async (req: AgencyApiRequest, res: AgencyApiRe
     // database write is crucial; failures of cache/stream should be noted and repaired
     const recorded_event = await db.writeEvent(event)
     try {
-      await Promise.all([cache.writeEvent(recorded_event), stream.writeEvent(recorded_event)])
+      await Promise.all([
```

Comment:
I feel like I see this pattern all over the place, we could probably make a `writeToDatastream()` helper
Edit: I mean as a follow up, out of scope here